dail/new-hire.vbs

```diff
@@ -199,6 +200,7 @@ DO
     	Dialog dialog1
     	cancel_confirmation
     	MAXIS_dialog_navigation
+		IF worker_signature = "" THEN err_msg = err_msg & vbCr & "* Please sign your case note."
```

Comment:
Same as NEW HIRE NDNH: Err_msg isn't set in the do...loop. No msgbox code to give the worker_signature error message.